sdk/core/azure-core/CMakeLists.txt

```diff
@@ -8,9 +8,13 @@ project(${TARGET_NAME} LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
+set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
+set(THREADS_PREFER_PTHREAD_FLAG TRUE)
+find_package(Threads REQUIRED)
```

Comment:
Is this a linux only package? If so, should this be under a non-windows ifdef?

Reply:
https://cmake.org/cmake/help/latest/module/FindThreads.html

It's **NOT** a linux only package. So it shouldn't under a non-windows ifdef.

Reply:
I understand the `Threads` package isn't windows only. But what about the `pthread` related CMAKE flags. Those are posix only, right? What do `CMAKE_THREAD_PREFER_PTHREAD` and `THREADS_PREFER_PTHREAD_FLAG` being true mean on Windows? Are they ignored or will they look for pthread specifically and fail? Presumably on windows, the call to `find_package` will set `CMAKE_USE_WIN32_THREADS_INIT`.

Reply:
whatever. It's a code change to core sdk. You guys decide what do with it.